src/coreclr-debug.ts

```diff
@@ -0,0 +1,245 @@
+'use strict';
```

Comment:
Need copyright header. Look at other .ts files in the repo to get the boilerplate.